src/main/java/hudson/plugins/libvirt/BeforeJobSnapshotJobProperty.java

```diff
@@ -54,7 +54,11 @@ public BeforeJobSnapshotJobProperty(EnableData snapshots) {
     }
 
     public String getSnapshotName() {
-        return getSnapshotsEnabled() ? snapshotsEnabled.getSnapshotName() : null;
+        if (!getSnapshotsEnabled()) {
+            return null;
```

Comment:
I'm always thinking that ternary operator is much simpler than the common if. But the result here is the same.